src/components/Class/ClassList/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from 'react-redux';
+import moment from 'moment';
```

Comment:
Could we put one blank line before our file imports? Somthing like this. 
```
import React, { useEffect } from "react";
import { useSelector, useDispatch } from 'react-redux';
import moment from 'moment';
// one blank line after package imports
import { actions as classActions } from '../redux/action'
import Loader from '../../common/Loader'
import "./styles.scss";

// 2 blank lines before our component code starts.
function ClassList() {
```

Reply:
done